src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToControlGeneratedAssemblyInfo.cs

```diff
@@ -370,6 +370,110 @@ public void It_includes_internals_visible_to()
             AssemblyInfo.Get(assemblyPath)["InternalsVisibleToAttribute"].Should().Be("Tests");
         }
 
+        [Fact]
+        public void Requires_preview_features_enabled()
+        {
+            TestPreviewFeatures(true, true);
+        }
+
+        [Fact]
+        public void Requires_preview_features_disabled()
+        {
+            TestPreviewFeatures(true, false);
+        }
+
+        [Fact]
+        public void Requires_preview_features_not_specified()
+        {
+            TestPreviewFeatures(false, false);
+        }
+
+        private void TestPreviewFeatures(bool addPreviewFeatureProperty, bool enablePreviewFeatures)
```

Comment:
Nit: Now that you're passing the right `identifier` in to `CopyTestAsset`, you can convert this back to a `Theory`.